docs/installation.md

```diff
@@ -104,9 +113,12 @@ mvn dependency:copy -Dartifact=org.transmartproject:transmart-server:17.1-SNAPSH
 java -jar transmart-server-17.1-SNAPSHOT.war
 ```
 
-## 6. Start services
 
-`Rserve` and `Solr` services can be run using `transmart-server` (to fetch `transmart-data`, see [Generate configuration files](#3-generate-configuration-files))
+## 5. Start services
+
+`Rserve` and `Solr` services can be run using `transmart-server` (to fetch `transmart-data`, 
```

Comment:
Did you mean using `transmart-data`?

Reply:
I didn't notice this typo since I actually skipped this part, will fix.